app/controllers/games_controller.rb

```diff
@@ -10,6 +10,7 @@ def new
 
   def create
     @game = Game.create(game_params)
+    #@game.white? == current_user
```

Comment:
The #white? method is in the piece model so won't actually work directly on @game here. I think we need something different, but I'm not exactly sure what. Maybe we could do `@game.white_player = current_user`? (Though we don't have a `:white_player` field yet in our games table.)

Reply:
Yeah, I didn't want to have it be active code until we get that figured out!